src/importexport/capella/internal/capella.cpp

```diff
@@ -839,13 +839,15 @@ static Fraction readCapVoice(Score* score, CapVoice* cvoice, int staffIdx, const
         case CapellaNoteObjectType::CLEF:
         {
             LOGD("     <Clef>");
+            static ClefType pclef = ClefType::INVALID;
```

Comment:
It looks really scary to use a static variable here. I think we should use a non-static variable that is created somewhere higher up in the call hierarchy, and then passed as a parameter to the called functions that need it. Basically, just like we had done here: 3a7c0e7b84184f85f6ca7e15379c0b9e8107d042

With static variables, I'm afraid there will be wrong results in multi-staff files, and when simply importing multiple files in succession.

Reply:
I can't believe that O made the same mistake again, after such a short time :-(

However: these first clefs seem to be superfluous too, if they are the default clef for the instrument, arent't they? I just can't find the condition to check there.